desktop/src/cli.rs

```diff
@@ -164,6 +164,12 @@ pub struct Opt {
     #[clap(long)]
     pub dummy_external_interface: bool,
 
+    /// Whether to allow downloading and using an external H.264 decoder binary.
+    ///
+    /// OpenH264 Video Codec provided by Cisco Systems, Inc.
+    #[clap(long, default_value = "true")]
+    pub enable_openh264: Option<bool>, // TODO: rename to "enable_external_video_backend" or something like that...?
```

Comment:
Regarding the TODO: we're probably going to have other video backends eventually, at the very least for supporting hardware decoding of H.263 or H.264 streams. "External video backend" wouldn't be descriptive enough in that case.